### PR TITLE
OpsCenter 6.8.33 Release

### PR DESCRIPTION
**Pull request by vimal-ds**
----
## Release Notes Automation
If you name your pull-request as "Product x.y.z Release ...", after merging the
PR, a GitHub Action will automatically create a product version tag "product-x.y.z".

Supported product names are:
 * DSE
 * OpsCenter
 * Studio
 * Luna Streaming

Version supports 3 sets or 4 sets of digits.

### CODE REVIEW

File: OpsCenter_6.8_Release_Notes.md [hunk 1]
```diff
@@ -1,5 +1,25 @@
 # Release notes for OpsCenter
 
+# Release Notes for OpsCenter 6.8.33
+
+## Backup Service
+* Encryption key collision during restore is now prevented and logged. Keys from system tables will no longer be included when restoring. (OPSC-17115)
```
Comment (tiagomlalves) [inline OpsCenter_6.8_Release_Notes.md:6]: Please rewrite according to https://datastax.jira.com/wiki/spaces/EN/pages/3374972929/Release+notes+guideline

File: OpsCenter_6.8_Release_Notes.md [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+
+## Core
+* Added gossip configuration files to diagnostics tarball. (OPSC-17480)
+* Added {{alert_on_no_host_available}} to the {{[cassandra]}} section of the cluster conf. When set to true an alert will be triggered if opscenterd is unable to make a cql connection to the monitored cluster. (OPSC-17398)
+
+## CVE
+* Added {{redirect_host}} config value to the {{[webserver]}} section of {{opscenterd.conf}} to mitigate host header injection vulnerability. (OPSC-17265)
```
Comment (tiagomlalves) [inline OpsCenter_6.8_Release_Notes.md:13]: This provides too much information allowing a potential attacker target older versions of OpsCenter. Also, it doesn't address a specific CVE (even though it's a security issue).
I propose to rewrite this under Core, stating that we added parameter `redirect_host` to force the hostname used in redirects - or something along those lines.

File: OpsCenter_6.8_Release_Notes.md [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+
+## Monitoring
+* Fixed issue in diagnostic tarball generation when {{diagnostic_tarball_skip_instance_check}} parameter is True. (OPSC-17471)
```
Comment (tiagomlalves) [inline OpsCenter_6.8_Release_Notes.md:16]: From a customer point of view, what was the issue or what did they observe before?

File: OpsCenter_6.8_Release_Notes.md [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+* Added the {{Swap}} value to the os-metrics memory.json file to identify if {{Swap}} is enabled from a diagnostics generated using OpsCenter. (OPSC-17405)
+* Resolved an issue that made it appear as if alerts were triggered on the configured threshold instead of over it. (OPSC-17446)
```
Comment (tiagomlalves) [inline OpsCenter_6.8_Release_Notes.md:18]: Same comment as before. This fix was a super specific targeting `os-disk-usage`. Why not let the customer know about it?

**Review by tiagomlalves — CHANGES_REQUESTED** [review]

File: OpsCenter_6.8_Release_Notes.md (hunk 3 above)
Comment (vimal-ds) [reply OpsCenter_6.8_Release_Notes.md:16]: The customer was unable to generate a diagnostic tarball. They had issues with instance_check, so we introduced this new parameter  {{diagnostic_tarball_skip_instance_check}} in the previous release to skip instance check before generating a diagnostic tarball. But it had issues reading this parameter values from config file. We fixed it now for this release.

File: OpsCenter_6.8_Release_Notes.md (hunk 4 above)
Comment (vimal-ds) [reply OpsCenter_6.8_Release_Notes.md:16]: Although the jira is specific to `os-disk-usage`. but fix we made is generic for all metrics.

Comment (tiagomlalves) [reply OpsCenter_6.8_Release_Notes.md:17]: What about:
> Fixed metrics computation causing alerts to be triggered when values were close but below their threshold.

File: OpsCenter_6.8_Release_Notes.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+* Added `{{alert_on_no_host_available}}` to the `{{[cassandra]}}` section of the cluster conf. When set to true an alert will be triggered if opscenterd is unable to make a cql connection to the monitored cluster. (OPSC-17398)
+* Added `{{redirect_host}}` config value to the `{{[webserver]}}` section of `{{opscenterd.conf}}` to force the hostname used in redirects. (OPSC-17265)
+
+## Monitoring
+* Fixed issue in diagnostic tarball generation when `{{diagnostic_tarball_skip_instance_check}}` parameter is True. (OPSC-17471)
+* Added the `{{Swap}}` value to the os-metrics memory.json file to identify if `{{Swap}}` is enabled from a diagnostics generated using OpsCenter. (OPSC-17405)
+* Resolved an issue that made it appear as if alerts were triggered on the configured threshold instead of over it. (OPSC-17446)
+
+## Repair Service
+* The repair service will now automatically ignore tables using the time window compaction strategy. (OPSC-17449)
```
Comment (tiagomlalves) [inline OpsCenter_6.8_Release_Notes.md:20]: what about:
> Improved repair service to automatically ...

**Review by tiagomlalves — APPROVED** [review]
I've provided suggestions for improvement. No need for another review round.

**Pull request merged**